### PR TITLE
added lalitpur city

### PR DESCRIPTION
**Pull request by bntshakya**
## What kind of change does this PR introduce?

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added a new city
- [ ] Other(s)

### CODE REVIEW

**Review by drishyakarki — APPROVED** [review]
LGTM

File: data/borderlines.json [hunk 1]
```diff
@@ -184,6 +184,47 @@
...
+        {
+          "name":"Rudra Varna Mahavihar",
+          "longitude":27.6681,
+          "latitude":85.3271
+        }
+        {
+          "name":"Labim Mall",
+          "longitude":27.6771,
+          "latitude":85.3171
+        }
```
Comment (thenishantsapkota) [inline data/borderlines.json:225]: Include atleast 5 landmarks per city please

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

**Comment by bntshakya** [comment]
@thenishantsapkota requested updates are made

**Review by thenishantsapkota — APPROVED** [review]
LGTM!
Thank you for your contribution @bntshakya

**Pull request merged**